pkg/runtime/runtime_test.go

```diff
@@ -115,7 +129,8 @@ func TestInitPubSub(t *testing.T) {
 		fakeReq.WithRawData(nil, "application/json")
 
 		fakeResp := invokev1.NewInvokeMethodResponse(200, "OK", nil)
-		fakeResp.WithRawData([]byte("[ \"topic0\", \"topic1\" ]"), "application/json")
+		subs := getSubscriptionsJSONString([]string{"topic0", "topic1"})
+		fakeResp.WithRawData([]byte(subs), "application/json")
```

Comment:
This unit-tests is testing only when topic == route. I think we need to add the test when topic != route.

Reply:
Added